akka-persistence-typed/src/main/scala/akka/persistence/typed/serialization/ReplicatedEventSourcingSerializer.scala

```diff
@@ -82,6 +83,8 @@ import scala.collection.immutable.TreeMap
 
   private val PublishedEventManifest = "PA"
 
+  private val TaggedManifest = "TA"
```

Comment:
I would prefer to have serialization for Tagged in `akka-persistence/src/main/scala/akka/persistence/serialization/MessageSerializer.scala`

Even though we currently only need it for ReplicatedEventSourcing it would be difficult to move if we need it later in some other module.